src/util/std_expr.h

```diff
@@ -44,16 +38,11 @@ class transt:public exprt
   const exprt &trans() const { return op2(); }
 };
 
-/*! \brief Cast a generic exprt to a \ref transt
- *
- * This is an unchecked conversion. \a expr must be known to be \ref
- * transt.
- *
- * \param expr Source expression
- * \return Object of type \ref transt
- *
-
-*/
+/// Cast a generic exprt to a \ref transt
+/// This is an unchecked conversion. \a expr must be known to be \ref
```

Comment:
\a ?

Reply:
I'd rather say what the behaviour is when it is not a transt.

Reply:
`\a` is to emphasize arguments.

Reply:
If it's not a `transt` then we have an invariant violation. Seems self-explanatory to me. What would you want to write?

Reply:
Presumably @peterschrammel means something like "If \a expr is not a \ref transt then an exception is thrown" (or whatever actually happens).

Reply:
There's only a PRECONDITION on the input param. I can't think of a better documentation than what is already there. There's no special handling in case the argument is not a `transt`.

Reply:
My comment was wrt the term 'unchecked' which might not be clear enough.

Reply:
I removed all mentions of "unchecked" for now, as it's a bit confusing.